### PR TITLE
DOCAPI-3697: Update of GraphiteMergeTree description.

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):
- Documentation

### CODE REVIEW

File: docs/en/operations/table_engines/graphitemergetree.md [hunk 1]
```diff
@@ -75,6 +75,23 @@ Rollup configuration structure:
 
 ```
 required-columns
+patterns
+```
+
+### Required columns
```
Comment (blinkov) [inline docs/en/operations/table_engines/graphitemergetree.md]: **C**olumns

File: docs/en/operations/table_engines/graphitemergetree.md [hunk 2]
```diff
@@ -75,6 +75,23 @@ Rollup configuration structure:
...
+
+Set the following parameters only if the name of columns differs from their default values:
```
Comment (blinkov) [inline docs/en/operations/table_engines/graphitemergetree.md]: Needs some extra introduction: these columns have to be present in a table to make it work and it's imporant, while the settings to override their names are kind of secondary and could be introduced later.

File: docs/en/operations/table_engines/graphitemergetree.md [hunk 3]
```diff
@@ -95,14 +112,16 @@ default
     ...
 ```
 
-**Important:** The order of patterns should be next:
+!!! note "Attention"
+    Patterns must be strictly ordered. Read about it below.
```
Comment (blinkov) [inline docs/en/operations/table_engines/graphitemergetree.md]: * I'd just put the order list right inside attention block and remove the "below" reference
* note -> warning

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/en/operations/table_engines/graphitemergetree.md (hunk 2 above)
Comment (BayoNet) [reply docs/en/operations/table_engines/graphitemergetree.md]: This introduction is in `Creating a Table` section.

**Comment by BayoNet** [comment]
Тут вроде всё хорошо. Вмёржить бы :-).

**Pull request merged**